src/devtools/client/debugger/src/components/Editor/Breakpoints/Breakpoint.js

```diff
@@ -11,11 +11,25 @@ import { connect } from "devtools/client/debugger/src/utils/connect";
 import { getDocument, toEditorLine } from "devtools/client/debugger/src/utils/editor";
 import { features } from "devtools/client/debugger/src/utils/prefs";
 import { resizeBreakpointGutter } from "devtools/client/debugger/src/utils/ui";
-import { isBreakable, isLogpoint } from "../../../utils/breakpoint";
-
-const breakpointSvg = document.createElement("div");
-breakpointSvg.innerHTML =
-  '<svg viewBox="0 0 60 15" width="60" height="15"><path d="M53.07.5H1.5c-.54 0-1 .46-1 1v12c0 .54.46 1 1 1h51.57c.58 0 1.15-.26 1.53-.7l4.7-6.3-4.7-6.3c-.38-.44-.95-.7-1.53-.7z"/></svg>';
+import { isBreakable } from "../../../utils/breakpoint";
+
+const BreakpointMarker = ({ onContextMenu, onClick, onMouseEnter, onMouseLeave }) => {
+  return (
+    <div
+      className={classnames("editor new-breakpoint", {
+        "folding-enabled": features.codeFolding,
+      })}
+      onContextMenu={onContextMenu}
+      onClick={onClick}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+    >
+      <svg viewBox="0 0 60 15" width="60" height="15">
```

Comment:
What do you think about pulling this into a little named component?

Reply:
I think this was this way for a reason, but I don't remember :)

It's probably safe to do this, but it could also cause some performance issues. For historical reasons, why are you changing this here and is it possible to do this in it's own commit so it's easier to revert if needed or compare quickly

Reply:
I was running into a problem earlier where this file was being rendered serverside where the window was unavailable. We don't have that problem anymore so I could just undo it.